bundle/manifests/loki-operator.clusterserviceversion.yaml

```diff
@@ -167,16 +225,22 @@ spec:
                   periodSeconds: 10
                 resources:
                   limits:
-                    cpu: 100m
-                    memory: 30Mi
+                    cpu: 200m
+                    memory: 128Mi
                   requests:
                     cpu: 100m
-                    memory: 20Mi
+                    memory: 64Mi
```

Comment:
I think we need to remove `resources` from the operator altogether. See https://github.com/openshift/elasticsearch-operator/pull/683/files for reference.